diff --git a/node_modules/manifesto.js/src/Canvas.ts b/node_modules/manifesto.js/src/Canvas.ts
index 4bacf31..5923650 100644
--- a/node_modules/manifesto.js/src/Canvas.ts
+++ b/node_modules/manifesto.js/src/Canvas.ts
@@ -1,3 +1,4 @@
+import { ViewingHint } from "@iiif/vocabulary/dist-commonjs";
 import {
   Annotation,
   AnnotationList,
@@ -258,4 +259,8 @@ export class Canvas extends Resource {
   getHeight(): number {
     return this.getProperty("height");
   }
+
+  getViewingHint(): ViewingHint | null {
+    return this.getProperty("viewingHint");
+  }
 }
